test/functional/cmdLineTests/CDSAdaptorTest/readme.txt

```diff
@@ -25,11 +47,11 @@ Few guidelines/assumptions when writing/updating tests placed in this project:
 	|
 	|--org.eclipse.osgi_<version>.jar: this is Equinox jar file, an OSGi framework implementation.
 	|
-	|--Test1Bundles: this directory contains bundles required by java class Test1.
+	|--TestBundle: this directory contains bundle source code required by java class TestBundle.
```

Comment:
In line 48, `org.eclipse.osgi_<version>.jar: ......`, probably you can mention cds adaptor is inside this jar.

Reply:
Added the information that this jar contains cds adaptor.